desc/compute/_profiles.py

```diff
@@ -607,6 +607,97 @@ def _iota_rr(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="iota_current",
+    label="\\iota_{\\mathrm{current}}",
+    units="~",
+    units_long="None",
+    description="Rotational transform (normalized by 2pi), current contribution",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["iota_den", "iota_current_num"],
+)
+def _iota_current(params, transforms, profiles, data, **kwargs):
+    data["iota_current"] = transforms["grid"].replace_at_axis(
+        data["iota_current_num"] / data["iota_den"], 0
+    )
+    return data
+
+
+@register_compute_fun(
+    name="iota_vacuum",
+    label="\\iota_{\\mathrm{vacuum}}",
+    units="~",
+    units_long="None",
+    description="Rotational transform (normalized by 2pi), vacuum contribution",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["iota_den", "iota_vacuum_num"],
+    axis_limit_data=["iota_den_r", "iota_num_r"],
+)
+def _iota_vacuum(params, transforms, profiles, data, **kwargs):
+    data["iota_vacuum"] = transforms["grid"].replace_at_axis(
+        data["iota_vacuum_num"] / data["iota_den"],
+        lambda: data["iota_num_r"] / data["iota_den_r"],
+    )
+    return data
+
+
+@register_compute_fun(
+    name="iota_current_num",
+    label="\\iota_{\\mathrm{current, numerator}}",
+    units="m^{-1}",
+    units_long="inverse meters",
+    description="Numerator of rotational transform formula, current contribution",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["current", "psi_r"],
+)
+def _iota_current_num(params, transforms, profiles, data, **kwargs):
+    """Current contribution to the numerator of rotational transform formula."""
+    # 4π^2 I = 4π^2 (mu_0 current / 2π) = 2π mu_0 current
+    data["iota_current_num"] = 2 * jnp.pi * mu_0 * data["current"] / data["psi_r"]
+    data["iota_current_num"] = transforms["grid"].replace_at_axis(
+        data["iota_current_num"], 0
+    )
+    return data
+
+
+@register_compute_fun(
+    name="iota_vacuum_num",
+    label="\\iota_{\\mathrm{vacuum, numerator}}",
+    units="m^{-1}",
+    units_long="inverse meters",
+    description="Numerator of rotational transform formula, vacuum contribution",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["lambda_z", "g_tt", "lambda_t", "g_tz", "sqrt(g)"],
+)
+def _iota_vacuum_num(params, transforms, profiles, data, **kwargs):
+    """Vacuum contribution to the numerator of rotational transform formula."""
+    data["iota_vacuum_num"] = surface_integrals(
+        transforms["grid"],
+        (data["lambda_z"] * data["g_tt"] - (1 + data["lambda_t"]) * data["g_tz"])
+        / data["sqrt(g)"],
+    )
+    data["iota_vacuum_num"] = transforms["grid"].replace_at_axis(
+        data["iota_vacuum_num"], 0
+    )
```

Comment:
I'm not certain what the proper limit should be here. It probably should not be 0, but it doesn't really matter because the proper limit get used for `iota_vacuum` which is what we ultimately care about.

Reply:
It's fine to put as 0. I think we're basically just assuming that at the time this function is called the configuration is in a physical state (current_r = 0 at axis).
Do you want to add a comment like this:
```python
# Assume current may be expanded as an even power series of ρ so that
# ∂ᵨ current = 0 at axis. Then 0 = lim iota_num = lim iota_current_num + lim iota_vacuum_num
# and the latter limit is also finite because the integrand is finite over the magnetic axis.
```

Reply:
The (non-zero) correct axis limit should be implemented now.